dev-docs/bidders/adinify.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: bidder
+title: Adinify
+description: Adinify Bidder Adapter
+biddercode: adinify
+aliasCode : smarthub
+usp_supported: true
+coppa_supported: true
+schain_supported: true
+dchain_supported: true
+media_types: banner, video, native
+safeframes_ok: true
+deals_supported: true
+floors_supported: true
+fpd_supported: false
+pbjs: true
+pbs: true
```

Comment:
```suggestion
pbs: false
```

You can speed up merging by chaning this to false - then we can merge and publish it.
Open a second PR changing this flag to `true` again, linked to the Prebid Server PR.

Prebid Server's release schedule is not as rigid as the JS one.

Reply:
ok I changed pbs to false